### PR TITLE
Ensure getDelimitedInput handles trailing delimiters correctly

### LINKED ISSUE #59: Ensure getDelimitedInput handles trailing delimiters correctly
## Expected Behavior

`getDelimitedInput` should ignore trailing delimiters (i.e. trailing comma or newline)

## Current Behavior

Unknown

## Possible Solution

Add a unit test for this case and ensure behaviour is expected

### PR DESCRIPTION
**Pull request by cooperbenson-qz**
## Description

Removed empty elements from the output of `getDelimitedInput`

## Motivation and Context

Fixes #59

## How Has This Been Tested?

Added new unit tests

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] ~My change requires a change to the documentation.~
- [ ] ~I have updated the documentation accordingly.~
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/utils/inputs.ts [hunk 1]
```diff
@@ -35,5 +35,5 @@ export function getDelimitedArrayInput(name: string, options?: InputOptions): st
   const value = getInput(name, options);
   const sep = _.includes(value, '\n') ? '\n' : ',';
   // eslint-disable-next-line @typescript-eslint/unbound-method
-  return _.map(_.split(value, sep), _.trim);
+  return _.map(_.filter(_.split(value, sep)), _.trim);
```
Comment (karoun) [inline src/utils/inputs.ts:38]: Could we instead run the filter after trimming? And add a test for inputs with spaces between their commas?

**Review by karoun — APPROVED** [review]
Nice! 👍 

Side note: is there a way to compile `dist/` at build/tagging time instead? Committing it to the repo works just fine, though it is a bit of anti-pattern.

**Comment by cooperbenson-qz** [comment]
> Side note: is there a way to compile `dist/` at build/tagging time instead? Committing it to the repo works just fine, though it is a bit of anti-pattern.

The build has to be run for the functional tests to work correctly. This also seems to be standard practice in all of Github's actions repos (including the template I based this one off).

I agree it's kind of gross, but this is the best pattern I've seen so far

File: src/utils/inputs.ts (hunk 1 above)
Comment (cooperbenson-qz) [reply src/utils/inputs.ts:38]: Already have a test for that (see [this test](https://github.com/quizlet/commit-changes-action/blob/46a940e46b81b70f404e962712df13d1117d26b5/src%2Futils%2F__tests__%2Finputs.test.ts#L40)), but I'll reorder the operations 😄

Comment (karoun) [reply src/utils/inputs.ts]: If there's already a test, how's it passing? Also `comman` -> `comma` in the test name 😁

**Pull request merged**